sdk/src/test/java/com/qonversion/android/sdk/storage/UserPropertiesStorageTest.kt

```diff
@@ -38,4 +44,66 @@ class UserPropertiesStorageTest {
         "any_string_key_2" to "any_string_value_2"))
         Assert.assertTrue(userPropertiesStorage.getProperties().isEmpty())
     }
+
+    @Test
+    fun `get handled properties when storage contains valid value`() {
+        // given
+        val properties = mapOf("someKey" to "someValue")
+        val jsonString: String = JSONObject(properties).toString()
+
+        every {
+            mockSharedPreferencesCache.getString("com.qonversion.keys.handled_properties_key", defValue = null)
```

Comment:
Maybe worth everywhere in mocks instead of passing concrete `defValue` replace it with `any()`? Then changing it in implementation won't break tests